SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -352,7 +352,7 @@ UClass* USpatialClassInfoManager::GetClassByComponentId(Worker_ComponentId Compo
 	}
 	else
 	{
-		UE_LOG(LogSpatialClassInfoManager, Warning,
+		UE_LOG(LogSpatialClassInfoManager, Log,
```

Comment:
I confirmed with @nafonso that he changed this from `Warning` to `Log` because it is expected to sometimes trigger as part of the tests.